ngx-fudis/projects/ngx-fudis/src/lib/components/form/select/multiselect/multiselect-chip-list/multiselect-chip-list.component.html

```diff
@@ -1,13 +1,18 @@
 <ul #chipListRef class="fudis-multiselect-chip-list" [attr.id]="parentId + '-selected-items'">
-  <button
-    *ngFor="let item of selectedItems; let index = index"
-    class="fudis-multiselect-chip-list__button"
-    [attr.aria-describedby]="parentId + '-remove-items'"
-    (focus)="_setFocus(true)"
-    (blur)="_setFocus(false)"
-    (click)="_clickChip(item, index)"
-  >
-    <fudis-icon [icon]="'close'" [color]="'white'"></fudis-icon>
-    {{ item.label }}
-  </button>
+  <ng-container *ngFor="let item of selectedItems; let index = index">
+    <li class="fudis-multiselect-chip-list__item">
+      <button
+        class="fudis-multiselect-chip-list__button"
```

Comment:
Also, now that we have 'fudis-multiselect-chip-list__item' class as li parent, let's follow BEM syntax and rename this button class to 'fudis-multiselect-chip-list__item__button'.